palaeontology.csl

```diff
@@ -106,8 +106,12 @@
             <group suffix=".">
               <group>
                 <text variable="title" font-style="italic" text-case="capitalize-first" suffix=". "/>
-                <text value="In" font-style="italic" suffix=" "/>
-                <text macro="editor" suffix=" "/>
+                <choose>
+                  <if variable="editor">
+                    <text value="In" font-style="italic" suffix=" "/>
+                    <text macro="editor" suffix=" "/>
+                  </if>
+                </choose>
                 <group>
```

Comment:
Let's do this properly:
```<group delimiter=" " suffix=" ">
  <text term="in" text-case="capitalize-first" font-style="italic"/>
  <text macro="editor">
</group>```
(if a group contains a variable, it only gets printed if that variable is present; also, terms are preferable to "text value" for versatility of styles down the road).